utils/log/logger.go

```diff
@@ -122,10 +125,11 @@ var prefixStyles = map[LevelType]struct {
 	color    color.Color
 	emoji    string
 }{
-	DEBUG: {logLevel: "Debug", color: color.Cyan},
-	INFO:  {logLevel: "Info", emoji: "🔵", color: color.Blue},
-	WARN:  {logLevel: "Warn", emoji: "🟠", color: color.Yellow},
-	ERROR: {logLevel: "Error", emoji: "🚨", color: color.Red},
+	VERBOSE: {logLevel: "Verbose", color: color.Gray},
```

Comment:
Does the length of `VERBOSE` string mess with way the logs look? Is there some padding added to log level to align fields?